test/apps/autoplot_app.py

```diff
@@ -151,9 +155,13 @@ def main(dataSrc):
         QtWidgets.QApplication.instance().exec_()
 
 
+# plotWidgetClass = MPLAutoPlot
+plotWidgetClass = PGAutoPlot
```

Comment:
woudl be cool at some point to convert this "manual" test into automated and parametrize over the plot widget class :)

Reply:
True :) for now it does the job, but i should probably come up with something better (like a command line argument or so)